config/sources/families/bcm2711.conf

```diff
@@ -79,7 +79,7 @@ extension_prepare_config__prepare_rpi_flash_kernel() {
 	declare -g RASPI_DISTRO_KERNEL="${RASPI_DISTRO_KERNEL:-no}" # Include a distro-built kernel?
 	declare -g SERIALCON="${RASPI_SERIALCON:-tty1}"             # HDMI etc, not serial. most people don't have UART on rpi
 
-	local usable_releases="jammy|kinetic|lunar"
+	local usable_releases="jammy|kinetic|lunar|mantic"
```

Comment:
@rpardini can we do something with these lists? every time in a bunch of lines we have to add/remove values.
maybe put the definitions in one file?

Reply:
In different places, these lists are different, depending on the context.

Reply:
Yeah, my idea is to create some kind of `releases.yaml` file where we can define everything that is release related, and  generate bash definitions from it. I'm hoping one day I can also get rid of the 'aggregation' stuff with similar things.